docs/build-n-test.md

```diff
@@ -14,11 +14,13 @@
 * dcurl allows various combinations of build configurations to fit final use scenarios.
 * You can execute `make config` and then edit file `build/local.mk` for custom build options.
     - ``BUILD_AVX``: build Intel AVX-accelerated Curl backend.
+    - ``BUILD_SSE``: build Intel SSE-accelerated Curl backend.
+    - ``BUILD_CPU``: build CPU Curl backend without acceleration.
```

Comment:
I think `BUILD_CPU` is a bit misleading. How about changing it to `BUILD_GENERIC` (generic C backend) or `BUILD_PORTABLE` (portable C backend)?

Reply:
It is confusing since `BUILD_AVX` and `BUILD_SSE` are based on `CPU`, too.
I chose `BUILD_GENERIC` to replace it.